app/controllers/users/induction_coordinators/registrations_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class Users::InductionCoordinators::RegistrationsController < Devise::RegistrationsController
```

Comment:
I think there's more potential to reuse code and avoid duplication if we organise this by feature rather than by type of user. I'd have it as Users::Registrations::InductionCoordinatorsController. This will add the endpoints specifically required by InductionCoordinators and anything shared between profiles would be in Users::RegistrationsController. What do you think?

Reply:
That sounds like a good idea